### ISSUE TITLE
Add issue templates for new documentations and translation

### ISSUE THREAD

**Issue by GindaChen** [wuhan2020/wuhan2020#343]
## 描述功能要求

The workflow shall be defined as follows:
1. (New Doc) Any new documentation that need multilingual support should create an **epic document issue** indicating that this document needs translation and will need to put into the website.
2. (Source Doc) Then the contributor who post this epic issue will open the **source document issue** and post the document in the issue body. This is as if we're using the issue page as the editor to edit and update the document.
3. (Translation) For any subsequent translation of this document, the translator should create a **translation document issue** and link this issue to the source document issue and epic document issue so that we can track the translation progress.
4. **(Doc change) Need help with this!!** 

## 额外的信息

Need discussion **when document changes**

**Comment by GindaChen** [comment]
@chenrui333 @atena118

I think that's somehow what we discussed today. 

I am a little not sure about point 4 - when the source document needs changes, do we:
1. Raise a new issue about the change and link this to the epic page (and hope people can pick up this change)
2. Raise `num_langauge` issue pages for the changes (and hope our issue poster have that patient)? 

Is there anyway to automate the process of change in document if we indeed use issue page?

**Comment by GindaChen** [comment]
@chenrui333 Also - I'm not sure I can write template issue using the github template editor. I guess I will need to raise the PR to add a new issue template. Would love you to review [this PR](https://github.com/wuhan2020/.github/pull/4) when you're free.

**Comment by chenrui333** [comment]
> @chenrui333 Also - I'm not sure I can write template issue using the github template editor. I guess I will need to raise the PR to add a new issue template. Would love you to review [this PR](https://github.com/wuhan2020/.github/pull/4) when you're free.

You should be able to just add issue_template into the github website repo. Let me test that.

The template is more for general usage though.

**Comment by GindaChen** [comment]
I think I don't have the permission to do that.

**Comment by GindaChen** [comment]
@Atena1118
我不太理解 DOCUMENTAZIONE 为啥不能是 DOCS ?

**Comment by GindaChen** [comment]
Yeah - I know issue is usually for general purpose. But since we want translator to engage in the repo, we probably need to provide a formal template for them to fill in (-- so we can be a lot easier...)

**Comment by chenrui333** [comment]
I have just merged the add-new-language support one, let's iterate on it.

**Comment by GindaChen** [comment]
@chenrui333 I actually can't see the issue template. I have doubt whether this is the correct way to add issue template on Github

![image](https://user-images.githubusercontent.com/32371474/74092267-11ece680-4a87-11ea-9166-8e34e38dba55.png)

**Comment by chenrui333** [comment]
@GindaChen 你这是在哪个下面打开的?

**Comment by chenrui333** [comment]
你的文件名弄错了。需要是markdown.

**Comment by GindaChen** [comment]
Ah I see...

**Comment by chenrui333** [comment]
我们社区基本上都是中国人，我觉得最好是不要用英文模板的 issue template。

anyway，template 的问题已经解决了。
![image](https://user-images.githubusercontent.com/1580956/74092774-c17d8500-4a96-11ea-8579-de880b5ae8a4.png)
